semmc/src/SemMC/Util.hs

```diff
@@ -243,7 +244,7 @@ exprToGroundVal :: forall sym tp.
                 -> S.SymExpr sym tp
                 -> Maybe (GE.GroundValue tp)
 exprToGroundVal BaseBoolRepr                 e = S.asConstantPred e
-exprToGroundVal (BaseBVRepr _w)              e = S.asSignedBV e
+exprToGroundVal (BaseBVRepr w)               e = S.asBV e
```

Comment:
Is this `w` unused?

Reply:
Yep, it is unused. Not sure where the underscore went, but I'll put it back.